### DISCUSSION TITLE
Pimcore 11 Update: How to read yaml files from symfony-config

### DISCUSSION THREAD

**Discussion by khenow1978** [pimcore/pimcore#16098] [General]
Hi.. I have just updated our running Pimcore installation to v11 and I'm facing some issues regarding the symfony-configs.

We used to create everything in YAML files so I wanted to go ahead and keep on using them. So I have set all new configurations to symfony-config with a proper directory in var/config/... (assetmetadata Definitions, Thumbnail, configs, etc.)

However in Pimcore they are not listed.. all I see in the documentation is the hint "needs container rebuild".. I'm not sure what that means. For the datahub I know I have to execute the "workspaces-rebuild" command but how about thumbnails, perspectives and the other settings? How do I rebuild the container??

Your help is much appreciated.

BR,
Karsten

**Comment by khenow1978** [comment]
It seems I had to clear the cache in order to rebuild the container.. after I executed this I'm now facing a new issue: HTTP 403 on the server. CLI seems to be working fine.. I can execute any command and don't get an error. Still when I want to open the pimcore website it responses with HTTP403, forbidden.

Those are the file permissions on the server: 
<img width="598" alt="image" src="https://github.com/pimcore/pimcore/assets/92732594/197394fc-36d8-459f-9230-e85692e1b4bd">

I found some information about permissions regarding the var/ and var/cache/ folder but I think they are all set properly. Furthermore it worked with the container before but just crashed after I rebuilt it. Any idea what could resolve this issue?

BR,
Karsten

**Comment by kingjia90** [comment]
Closing, as far as i see at the service desk ticket, this was caused by an abandoned htaccess file
